php/php-mysql-xdevapi/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               php 1.1
+
+name                    php-mysql-xdevapi
+set cap_name            mysql_xdevapi
+categories-append       net devel
+platforms               darwin freebsd openbsd
+maintainers             {ryandesign @ryandesign} openmaintainer
+license                 PHP-3.01
+categories              php devel
+
+homepage                https://dev.mysql.com/doc/x-devapi-userguide/en
+
+php.branches            7.1 7.2 7.3
+php.pecl                yes
+
+
+version                 8.0.15
+checksums               rmd160  563c690c4005e031db9905242d186043c82036d8 \
+                        sha256  ebafba923ec3a2f9a1cce7ba3c855338053a8ebf47f47c8fd66b6391029d8eed \
+                        size    443751
+
+distname                ${cap_name}-${version}
+
+description             MySQL X DevAPI for PHP
+
+long_description        The X DevAPI is the new common API for MySQL Connectors built on the X Protocol introduced in MySQL 5.7.12.
+
+depends_build-append    port:boost
+depends_lib-append      port:boost
```

Comment:
The `depends_build-append` line can be removed because it is included in `depends_lib-append`.

Reply:
I was not sure which once to include and how, I manages to build it from sources manually other day.

Reply:
Fixed Protobuf, building now.

Reply:
Latest changes are below.
https://github.com/macports/macports-ports/compare/b6e9c98cb4e629453b685525529adf4c01bca7be..a750bb075c2bf21772ba75cf2535702c2dc8a6ef

Reply:
@mf2k Apparently. php extension keep dynamic link to protobuf3.
Can you give a hint how to build it static, and remove dependencies if binary is installed?

Reply:
I'm not understanding what you are asking. We don't normally like static builds. We also don't generally work with non Macports dependencies. So adding a dependency on protobuf3-cpp is correct.